services/app-web/src/hooks/useStatePrograms.ts

```diff
@@ -0,0 +1,19 @@
+import { useAuth } from '../contexts/AuthContext'
+import { Program } from '../gen/gqlClient'
+
+// Get state programs from logged in state users data
+const useStatePrograms = (): Program[] | [] => {
+    const { loggedInUser } = useAuth()
+    let statePrograms: Program[] = []
+
+    if (loggedInUser && loggedInUser.__typename === 'StateUser') {
+        statePrograms = loggedInUser.state.programs
+    } else {
+        console.error(
```

Comment:
thanks for this. If it were more serious, coding errors are the kind of things I think it's appropriate to `throw` for.